dist/generator-vulcanjs/lib/ui-text.js

```diff
@@ -2,6 +2,7 @@ const chalk = require('chalk');
 
 const descriptions = {
   appName: 'The name of your app',
+  doShallowClone: 'Default is fast and great for testing. Full vulcan git clone will avoid requiring to unshallow the repository to change the git remote.',
```

Comment:
I know. Tried to be non technical: "fast vs. complete" and avoid the shallow as much as possible.

How would you automate unshallow?